SCENARIOS.md

```diff
@@ -186,12 +186,15 @@ yield unregister("ad6a1337-748a-437b-9957-6895f24796a9")
 ### send_notification
 
 Send a notification to the push service. If an empty data payload is desired,
-`None` can be used for `data`.
+`None` can be used for `data`. If VAPID headers are desired, include a
+valid `claims` dict.
 
-**Arguments:** `endpoint_url`, `data`, `ttl`
+**Arguments:** `endpoint_url`, `data`, `ttl`, `claims`
 
 ```python
-yield send_notification('BIG_URL', 'SOME_DATA', 60)
+yield send_notification('BIG_URL', 'SOME_DATA', 60,
+{aud:"https://example.com", sub,"mailto:admin@example.com",
+exp:1459803905})
```

Comment:
This number is in what format? I assume if its not in the future, than our server will reject it?